From 31760d1f5e73caf2525859de581e11c73e405218 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Sat, 10 Feb 2018 09:18:50 -0500
Subject: [PATCH] Remove usage of kickstart.get_modules() (rhbz#1544075)

As of pykickstart 3.9, this method doesn't exist anymore, and usage
of this was removed in livecd-tools already.
---
 appcreate/appliance.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/appcreate/appliance.py b/appcreate/appliance.py
index 61f1c46..34971a3 100644
--- a/appcreate/appliance.py
+++ b/appcreate/appliance.py
@@ -66,7 +66,6 @@ class ApplianceImageCreator(ImageCreator):
 
         #additional modules to include
         self.modules = ["sym53c8xx", "aic7xxx", "mptspi"]
-        self.modules.extend(kickstart.get_modules(self.ks))
 
         # This determines which partition layout we'll be using
         self.bootloader = None
-- 
2.14.3

